projects/client-api/src/index.js

```diff
@@ -1303,6 +1306,71 @@ export function updateZoom(level) {
 }
 chainList.updateZoom = updateZoom;
 
+/**
+ * Subscribe to label change and exit events
+ * @function selectionUpdates
+ * @param {object} [cache] - An optional cache object (empty object) to use for caching label updates across subscriptions.
+ * @return {Subscription} A {@link Subscription} that can be used to stop reacting to label updates
+ * @example
+ * GraphistryJS(document.getElementById('viz'))
+ *     .pipe(
+ *          selectionUpdates,
+ *          tap((s) => {
+ *                 console.log(s);
+ *          }),
+ *     })
+ *     .subscribe();
+ */
+export function selectionUpdates(g) {
```

Comment:
should also export elsewhere similar to labels

Reply:
done.